### DISCUSSION TITLE
epub and mobi rendering problem

### DISCUSSION THREAD

**Discussion by pstrg** [sumatrapdfreader/sumatrapdf#2300] [Q&A] [answered]
I note two problems with epub and mobi rendering:
1) Sometimes (not always) font size isn't respected: parts of the text that should be displayed with a smaller font are not (however, both Sigil, which was used to edit it, and Kindle do display them correctly) - please see image:
[https://ibb.co/sPXWQ8k](url)
2) Sometimes (not always) the index does not display at all (again: checked OK in other environments.)
Is it perhaps because SumatraPDF's html rendering engine is picky?

**Comment by GitHubRulesOK** [comment, marked as answer]
Currentl I cant edit your question but can post your image
![image](https://user-images.githubusercontent.com/12754913/151703825-2d7bc28c-32e8-4add-a38f-b8e6b1051976.png)

The rendering engine in current 3.4 pre-release has been standardized to using MuPDF for rendering, so most ePub books will look significantly different but mobi/aws files may still use the older renderer. Thus I have to suggest you try a current **portable** copy from https://www.sumatrapdfreader.org/prerelease (keep your old one, until you decide to switch totally, I have both but installed is the current pre-release)

**Reply by pstrg** [reply]
Current pre-release build (version 14255) renders as expected - both body and index.
Thank you.
